### ISSUE TITLE
no trigger

### ISSUE THREAD

**Issue by atomikx** [michbeck100/pimatic-alarm#5]
hello,

if i configure the plugin with 3 contact sensors and restart pimatic only the first Event (from all 3 devices) is showing in the "trigger" variable and in the frontend of the device. Every other following Event is not triggering.

My Configuration:

` {
      "id": "alarm_system",
      "name": "Alarm system",
      "class": "AlarmSystem",
      "includes": [
        "contact-window1",
        "contact-window2",
        "contact-door"
      ]
    },
    {
      "id": "alarm_trigger",
      "name": "Alarm",
      "class": "AlarmSwitch"
 }`

**Comment by michbeck100** [comment]
This is by design. Once an alarm is triggered, it won't be triggered again. And the triggering device won't change either. But i think this isn't unusual for an alarm system. What would you expect? I would say, there can be only ONE device triggering an alarm.

**Comment by atomikx** [comment]
you are right! But if a alarm was triggered, how can i reset the alarm-system to detect the next event (e.g. from another device)?

**Comment by michbeck100** [comment]
You can use the AlarmSwitch class to switch an alarm off. This won't turn off the complete system, just the alarm. After turning the alarm off, it should be armed again.

**Comment by atomikx** [comment]
The AlarmSwitch does not (automatically) turn on when an alarm is triggered. So I do it by a rule. This solution works with the first alarm detection (after pimatic restart) - the AlarmSwitch is switched on. If I then switch the AlarmSwitch off again, no other event is detected. And the Trigger variable is not reseted.

**Comment by michbeck100** [comment]
This could be a bug. I have to test myself
